HandleAjaxCall.php

```diff
@@ -1,5 +1,17 @@
 <?php
-if(!defined('sugarEntry') || !sugarEntry) die('Not A Valid Entry Point');
+
+if (!defined('sugarEntry') || !sugarEntry || !defined('SUGAR_ENTRY') || !SUGAR_ENTRY) {
+    die('Not A Valid Entry Point');
+}
+if (defined('sugarEntry')) {
+    $deprecatedMessage = 'sugarEntry is deprecated use SUGAR_ENTRY instead';
```

Comment:
Since this is a debug message. Similar to 
`$GLOBALS['log']->fatal('...');`
So in this context, this doesn't need to translated. The entry point code is there to prevent un-authorised access to php files in SuiteCRM. Especially when mod rewrite is disabled. A users is not likely to see this message. It is intended for administrators and developers during deployments.